README.md

```diff
@@ -19,19 +19,24 @@ module "admin_tier" {
   github_team                 = "${var.github_team}"
   instance_type               = "${var.instance_type}"
   vpc_id                      = "${var.vpc_id}"
-  security_groups             = ["${var.security_groups}"]
+  name                        = "admin"
+  namespace                   = "${var.namespace}"
+  stage                       = "${var.stage}"
   subnets                     = ["${var.subnets}"]
-  associate_public_ip_address = "${var.associate_public_ip_address}"
+  zone_id                     = "${module.tf_domain.zone_id}"
```

Comment:
Please extend `README.md` with this dependency.

And use backquotes (``) for expressions like this `tf_hostname` in README.md